client-failover/client-failover-sample-hazelcast5/src/main/java/sample/com/hazelcast/clientfailover/ClientWithSsl.java

```diff
@@ -0,0 +1,127 @@
+
+public class ClientWithSsl {
+
+    public static void main(String[] args) throws Exception {
+
+        ClientFailoverConfig clientFailoverConfig = new ClientFailoverConfig(); // <2>
+        clientFailoverConfig.addClientConfig(getPrimaryClientConfig());
+        clientFailoverConfig.addClientConfig(getSecondaryClientConfig());
+
+        HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient(clientFailoverConfig);
+
+        System.out.println("Connection Successful!");
+
+        nonStopMapExample(client); // <3>
+
+        client.shutdown();
+
+        System.exit(0);
+
+    }
+
+    // <4>
+    private static ClientConfig getPrimaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+        Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client.truststore").toURI().getPath());
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+            .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+            .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
+        config.setClusterName("YOUR_CLUSTER_ID");
+        config.getConnectionStrategyConfig()
+            .getConnectionRetryConfig()
+            .setClusterConnectTimeoutMillis(10000); // <5>
+        return config;
+        }
+    
+    // <6>
+    private static ClientConfig getSecondaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+       Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client2.keystore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client2.truststore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+           .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+           .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
```

Comment:
I think we can remove this property from the code samples. As far as I know, the cloud team is using it for testing purposes, but I don't see the point of exposing it to the user.

Reply:
@mdumandag. Just tried removing this from the code and it no longer runs as it can't seem to find the cluster. Do we need to provide a hard-coded value?

Reply:
Are you trying to connect to UAT or DEV environment? Because you shouldn't be setting this value for production clusters (clusters created in cloud.hazelcast.com)

Reply:
I'm trying on UAT. Thanks for explaining. I'll remove this value now.